tools/docker/tests/test_flows.sh

```diff
@@ -72,23 +72,23 @@ send_CAPI_command() {
 }
 
 test_initial_ap_config() {
-    status "test initial autoconfig"
-
-    check_error=0
-    check docker exec repeater1 sh -c \
-        'grep -i -q "WSC Global authentication success" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
-    check docker exec repeater1 sh -c \
-        'grep -i -q "WSC Global authentication success" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
-    check docker exec repeater1 sh -c \
-        'grep -i -q "KWA (Key Wrap Auth) success" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
-    check docker exec repeater1 sh -c \
-        'grep -i -q "KWA (Key Wrap Auth) success" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
-    check docker exec repeater1 sh -c \
-        'grep -i -q "Controller configuration (WSC M2 Encrypted Settings)" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
-    check docker exec repeater1 sh -c \
-        'grep -i -q "Controller configuration (WSC M2 Encrypted Settings)" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
-
-    return $check_error
+#    status "test initial autoconfig"
+
+#     check_error=0
+#     check docker exec repeater1 sh -c \
+#         'grep -i -q "WSC Global authentication success" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
+#     check docker exec repeater1 sh -c \
+#         'grep -i -q "WSC Global authentication success" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
+#     check docker exec repeater1 sh -c \
+#         'grep -i -q "KWA (Key Wrap Auth) success" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
+#     check docker exec repeater1 sh -c \
+#         'grep -i -q "KWA (Key Wrap Auth) success" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
+#     check docker exec repeater1 sh -c \
+#         'grep -i -q "Controller configuration (WSC M2 Encrypted Settings)" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
+#     check docker exec repeater1 sh -c \
+#         'grep -i -q "Controller configuration (WSC M2 Encrypted Settings)" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
+    python3 ${rootdir}/tools/docker/tests/run_tests.py
```

Comment:
This will run all tests. You should pass an argument to run one specific test (as a transient measure until all tests are converted and we can just call run_tests.py directly).